src/include/xlist.h

```diff
@@ -207,7 +208,7 @@ class xlist {
     bool first = true;
     for (const auto &item : list) {
       if (!first) {
-        oss << ", ";
+        oss << std::string(", ");
```

Comment:
i think the right fix is to `#include <ostream>` in this header file.

Reply:
@tchaikov 
Will try, thanx.